CHANGES.rst

```diff
@@ -4,6 +4,9 @@ Changelog for zest.releaser
 9.0.0a2 (unreleased)
 --------------------
 
+- Ignore error output when calling `build`.
+  We only need to look at the exit code to see if it worked.
```

Comment:
```suggestion
- Ignore error output when calling `build`.
  We only need to look at the exit code to see if it worked.
  You can call zest.releaser with ``--verbose`` if you want 
  to see the possible warnings.
```

Reply:
Ah nice, I did not realise this.  Checked, and working.